applications/nrf_desktop/doc/dfu_mcumgr.rst

```diff
@@ -0,0 +1,88 @@
+.. _nrf_desktop_dfu_mcumgr:
+
+Device Firmware Upgrade MCUmgr module
+#####################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Device Firmware Upgrade MCUmgr module is responsible for performing a Device Firmware Upgrade (DFU) over Bluetooth® LE.
```

Comment:
`over Simple Management Protocol (SMP).`
The module does not rely on Bluetooth LE transport.

Reply:
You could inform that for example you can use the module with Bluetooth LE transport for OTA DFU support.